src/_gettsim/arbeitslosengeld_2/inputs.py

```diff
@@ -7,7 +7,8 @@
 
 @policy_input(start_date="2023-01-01")
 def bezug_im_vorjahr() -> bool:
-    """Whether the person received Arbeitslosengeld 2 / Bürgergeld in the previous year."""
+    """Whether the person received Arbeitslosengeld 2 / Bürgergeld in the previous
+    calendar year."""
```

Comment:
Is that correct? My guess would have been that it is the 12 months before application, but then I never cease to be surprised...